Makefile

```diff
@@ -66,6 +69,8 @@ dockerized-test:
 	@echo running make $(TARGET) inside a container
 	docker build --pull -t apm-integration-testing .
 
+	mkdir -p -m 777 "$(PWD)/$(JUNIT_RESULTS_DIR)"
+	chmod 777 "$(PWD)/$(JUNIT_RESULTS_DIR)"
```

Comment:
Out of interest, why the `chmod 777` after you already created the directory with `-m 777`?

Reply:
Thanks for the speedy review turnaround.

The `mkdir` didn't change perms if the directory already existed, the case when I tested locally.  I forgot to remove it after adding the `chmod` but I don't suppose it hurts.